packages/formation-react/src/components/Sidebar/Sidebar.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Sidebar
+name: Sidebar
+tags: sidebar, component
+---
+
+To use this component you will also need to use the <SidebarBlock />
+component located inside the <Sidebar /> component folder.
+These sidebar blocks are passed in as children to the <Sidebar />
+component and they then render the individual blocks of the sidebar.
+
+### Code:
+
+```javascript
+import Sidebar from '@department-of-veterans-affairs/formation-react/Sidebar';
+import SidebarBlock from '@department-of-veterans-affairs/formation-react/SidebarBlock';
+
+const firstSidebarBlock = {
+  heading: 'This is the heading',
+  content: <div>This is some content</div>,
+  headingLevel: 3,
+};
+
+<Sidebar>
+  <SidebarBlock
+    heading={firstSidebarBlock.heading}
+    content={firstSidebarBlock.content}
+    headingLevel={firstSidebarBlock.headingLevel}
+  />
+</Sidebar>;
```

Comment:
I love how this component uses composition rather than complex data structures passed as props! This is very readable

Reply:
No need to make it more complicated that needed :-)